package-lock.json

```diff
@@ -71,7 +69,7 @@
         "typescript": "^5.5.2"
       },
       "engines": {
-        "node": ">=20.15.0"
```

Comment:
In my understanding this engine declaretion doesn't update the node version

Reply:
I guess the node is embedded when we upload the CLI https://github.com/autifyhq/autify-cli/blob/main/.github/workflows/upload.yml
But not 100% sure TBH.

What I can tell you is that we still ship the CLI with node 18.

```
$ autify --version    
@autifyhq/autify-cli/0.46.0 wsl-x64 node-v18.20.1
```